distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -309,3 +309,50 @@ Here are some helpful suggestions for setting up data backups and recoveries:
 * Run a scheduler program that calls the backup URL on a timed basis.
```

Comment:
Should we indicate the support we are doing does not run on Windows? Seems like our instructions would only work for *NIX type environments, so we should be explicit there.